components/map/MapMarkers.tsx

```diff
@@ -31,6 +33,14 @@ const MapMarkers = ({ markersData }: Props) => {
     ])
   }, [markersData])
 
+  const handlePointPress = useCallback(
+    (event: OnPressEvent) => {
+      if (!event?.features || event.features.length === 0) return
+      onPointPress?.(event.features[0] as Feature<Point, GeoJsonProperties>)
```

Comment:
Consistent coding style suggest to write always positive conditions if possible, so:
```
if (event?.features && event?.features.length > 0) {
  onPointPress?.(event.features[0] as Feature<Point, GeoJsonProperties>)
}
```

We also try to limit usage of "if oneliners", because it's easy to miss it while reading the code. It's always better to use braces, even for one-line statement.
```
if (condition) {
   return
}
```